dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -205,8 +205,11 @@ https://workforus.theguardian.com/careers/product-engineering/
 						: '<!-- no canonical URL -->'
 				}
                 <meta charset="utf-8">
-
-                <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+				${
+					renderingTarget === 'Web'
+						? `<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">`
+						: `<meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no">`
```

Comment:
Do we want to add a comment here explaining why these are different?

Reply:
good idea! I'll add one.

Reply:
Basically just this PR description 😆